nodewatcher/modules/defaults/network_profile/defaults/wireless.py

```diff
@@ -163,6 +163,21 @@ def get_project_ssid(purpose, attribute='essid'):
                         'routing_protocols': routing_protocols,
                     },
                 )
+
+            # Create 802.11s mesh interface.
+            if 'no-wifi-mesh' and '802.11s-wifi-mesh' in network_profiles:
```

Comment:
This check doesn't seem right to me, so `if 'no-wifi-mesh'` will always evaluate to True so it will be ignored (you probably meant `if 'no-wifi-mesh' in network_profiles and ...`).

Also, I think it would be better to move that into the above if, so you have something like this:
```python
if 'no-wifi-mesh' not in network_profiles:
    if '802.11s-wifi-mesh' in network_profiles:
        # Configure 802.11s.
    else:
        # Configure ad-hoc mesh.
```